libvcx/src/proof.rs

```diff
@@ -880,17 +420,18 @@ pub mod tests {
     #[test]
     #[cfg(feature = "general_test")]
     fn test_update_state_with_message() {
-        let _setup = SetupAriesMocks::init();
+        let _setup = SetupStrictAriesMocks::init();
```

Comment:
When we are at it (restoring proof tests again) it would be cool if we used `MockBuilder` approach to explicitly enable mocking of indy proof verification (in function `validate_indy_proof` at `proof.rs`)